flo-freezer/pom.xml

```diff
@@ -30,5 +30,22 @@
       <groupId>com.typesafe</groupId>
       <artifactId>config</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.twitter</groupId>
+      <artifactId>chill_2.11</artifactId>
+      <version>0.9.2</version>
```

Comment:
It's an awkward dependency to have in the parent pom because then we should move the chill dependencies from the scala submodules as well into the parent pom and then we end up enumerating the scala versions in the parent pom... do we want that?

Reply:
Ok then.